src/Term/TermList.php

```diff
@@ -190,4 +190,13 @@ public function hasTerm( Term $term ) {
 			&& $this->terms[$term->getLanguageCode()]->equals( $term );
 	}
 
+	/**
+	 * Removes all terms from this list.
+	 *
+	 * @since 5.1
+	 */
+	public function clear() {
```

Comment:
Same here, there is `Fingerprint::setLabels` and `setDescriptions`, so this should not be needed. At least not at the moment. Did I missed a proposed use case?